test/kafo/hook_context_test.rb

```diff
@@ -43,5 +44,17 @@ module Kafo
         assert_equal context.module_enabled?('known_module'), true
       end
     end
+
+    describe "#app_option?" do
+      specify do
+        app = {'known_option' => 'option_arg'}
+        config = Minitest::Mock.new
+        kafo.expect :config, config
+        config.expect :app, app
+        app.expect :key?, :known_option
```

Comment:
Why is this needed? You already have a hash with `known_option` so there's no need to mock anything